feat: compile each file (#17)
vuestorefront/theme-utilities: README.md

Script reads files from multiple sources and copies them to destination directory. If files with the same path (relative to each source) exist in multiple sources, priority will be given to source defined later in the configuration.

> Files with `.vue`, `.ts`, `.js`, `.json`, and `.yml` extensions are additionally parsed using `ejs` library, allowing them to contain build-time variables like `<%= some.source.variable %>`. If you want to parse all extensions then set `parseAllExtensions` to `true` in the configuration.

Reviewer: ```suggestion
Files with `.vue`, `.ts`, `.js`, `.json`, and `.yml` extensions are additionally parsed using `ejs` library, allowing them to contain build-time variables like `<%= some.source.variable %>`. If you want to parse all extensions, add `parseAllExtensions: true` to the configuration.
```
